index.html

```diff
@@ -1,19 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+              <header class="header">
+                    <img class="karma_logo" src="img/karma-logo.svg" alt="karma logo"> 
+                        <!--navabar-->
+                        <nav> 
+                            <ul class="nav_list">
+                                <li class="nav_item">
```

Comment:
Currently you have:

```html
<ul class="nav-list">
  <li class="nav-item">
    <a class="nav-link" href="#">Blog</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="#">How it Works</a>
  </li>
</ul>
```

What I'm suggesting is:

```html
<ul>
  <li>
    <a href="#">Blog</a>
  </li>
  <li>
    <a href="#">How it Works</a>
  </li>
</ul>
```

You can still apply the styles you want in CSS. But this makes HTML more compact and readable. 

Usually, when you repeat the same thing over and over (styles for each navigation item in our case) - it's a good sign there is a way to simplify and reduce the code duplication.